applications/Unity.GrantManager/modules/Volo.BasicTheme/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic/Themes/Basic/Components/SideNavbar/Default.cshtml

```diff
@@ -29,8 +29,8 @@
                 var elementId = string.IsNullOrEmpty(menuItem.ElementId) ? string.Empty : $"id=\"{menuItem.ElementId}\"";
                 var cssClass = string.IsNullOrEmpty(menuItem.CssClass) ? string.Empty : menuItem.CssClass;
                 var disabled = menuItem.IsDisabled ? "disabled" : string.Empty;
-                var url = string.IsNullOrEmpty(menuItem.Url) ? "#" : Url.IsLocalUrl(menuItem.Url) ? Url.Content(menuItem.Url.EnsureStartsWith('~')) : menuItem.Url;
-                var isCurrentUrlActive = (currentPath.Split('/').ElementAt(1) == url.Split('/').ElementAt(1));
```

Comment:
It should be the ElementAt(1)
for keeping the active state for the nested routes as well.

Let's say for "/GrantApplication" and "/GrantApplication/Details", we need to keep the GrantApplication menu item active.

Reply:
After discussion with Nitheesh : Adding a temporary fix for now to enable the some of the identity menus - these are breaking currently when added - these menus are needed as enablers for the other work going on